Doc/library/base64.rst

```diff
@@ -46,7 +40,16 @@ package instead.
    Any :term:`bytes-like objects <bytes-like object>` are now accepted by all
    encoding and decoding functions in this module.  Ascii85/Base85 support added.
 
-The modern interface provides:
+
+.. _base_64_rfc_4648:
+
+RFC 4648 Encodings
+------------------
+
+The :rfc:`4648` encodings are suitable for encoding binary data so that it can be
+safely sent by email, used as parts of URLs, or included as part of an HTTP
+POST request.  The encoding algorithm is not the same as the
+:program:`uuencode` program.
```

Comment:
```suggestion
POST request.
```

uuencode is long-obsolescent (removed from Python itself too) and not directly related to base64, so a reference here likely adds as much confusion as any it clarifies at this point